FileReeder.java

```diff
@@ -0,0 +1,33 @@
+import java.io.File;
+import java.util.Scanner;
+import java.io.FileNotFoundException;
+
+public class FileReeder {
+    public static void main(String args[]) {
+
+        File file;
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        String[] inputs = input.split(" ");
+        String command = inputs[0];
+        String pattern = inputs[1];
+        String theFile = inputs[2];
+
+        file = new File(theFile);
+        if (command.equals("search")) {
```

Comment:
What will happen if the user did not enter a valid command, for example, wrote "searrch". Perhaps print som usage instructions.